improver/spotdata/spot_manipulation.py

```diff
@@ -0,0 +1,249 @@
+# -*- coding: utf-8 -*-
```

Comment:
The new files in this PR: improver/spotdata/spot_manipulation.py, improver/spotdata/utilities.py, improver_tests/spotdata/test_SpotManipulation.py and improver_tests/spotdata/test_utilities.py now have a licence that conflicts with master following https://github.com/metoppv/improver/pull/2001, so you'll need to update these licences in your PR.

Reply:
Done.